### PR TITLE
feat: Continue an existing amendment. Select Amendment Type. Select effective Date. Select approval

### PR DESCRIPTION
**Pull request by IgnatG**
### Changelog:
1. Continue with existing an amendment request
2. Ability to enter if an approval is required
3. Ability to select an amendment type
4. Ability to enter an effective date for automatic amendment

### CODE REVIEW

**Review by abhi-markan — CHANGES_REQUESTED** [review]
Few minor changes.

File: trade-finance-manager-ui/server/controllers/case/amendments/amendmentRequestDate.controller.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    return res.redirect('/not-found');
+  }
+};
+
+/**
+ * posts amendment request date when continue button clicked
+ * includes request date, user, creation timestamp and changes status
+ */
+const postAmendmentRequestDate = async (req, res) => {
+  const { facilityId, amendmentId } = req.params;
```
[the remark this answers is not in the document]
Comment (IgnatG) [reply trade-finance-manager-ui/server/controllers/case/amendments/amendmentRequestDate.controller.js:46]: As discussed, this is handled in the api file

File: trade-finance-manager-ui/server/controllers/case/amendments/amendmentEffectiveDate.controller.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      amendmentEffectiveDateMonth,
+      amendmentEffectiveDateYear,
+    });
+  } catch (err) {
+    console.error('Unable to get the amendment effective date page', { err });
+    return res.redirect('/not-found');
+  }
+};
+const postAmendmentEffectiveDate = async (req, res) => {
+  const { facilityId, amendmentId } = req.params;
```
[the remark this answers is not in the document]
Comment (IgnatG) [reply trade-finance-manager-ui/server/controllers/case/amendments/amendmentEffectiveDate.controller.js:41]: As discussed, this is handled in the api file

File: trade-finance-manager-ui/server/controllers/case/amendments/amendmentOptions.controller.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+      hasBeenIssued,
+    });
+  } catch (err) {
+    console.error('Unable to get the amendment options page', { err });
+    return res.redirect('/not-found');
+  }
+};
+
+const postAmendmentOptions = async (req, res) => {
+  const { facilityId, amendmentId } = req.params;
```
[the remark this answers is not in the document]
Comment (IgnatG) [reply trade-finance-manager-ui/server/controllers/case/amendments/amendmentOptions.controller.js:35]: As discussed, this is handled in the api file

**Review by abhi-markan — APPROVED** [review]

**Pull request merged**